scripts/lint

```diff
@@ -37,12 +75,13 @@ lint=$($vimdir/bin/vim -esNR \
   +"e $vimgodir/doc/vim-go.txt" \
   +'verbose VimhelpLintEcho' \
   +q \
-  2>&1)
+  2>&1 ||:)
```

Comment:
Why is this change being made?

Reply:
To make sure the command is considered "tested" and the script doesn't exit with `set -e`.

Reply:
I assumed that :-). But why *wouldn't* we want it to exit non-zero in that case? Appending `||:` here will make it always exit with a successful zero status code, and if for some reason there weren't any output either, the script would exit zero and there would be no indication that something was wrong.

Reply:
The issue was that the script would stop if one linter fails, rather than continuing to the next.

To do this prettier is kind of where you hit the limits of shell scripting :-/ I can fix it if you really think it's worth it, but IMHO it's not.

Reply:
I don't understand what suddenly made `vim` exit non-zero, though; before these changes, the exit code of vim wasn't dependent on whether or not `vimhelplint` found problems.

Reply:
Maybe this specific linter never exits with `>0` on failures actually; I just copy/pasted this from the other two I added.